selfdrive/navd/main.cc

```diff
@@ -12,6 +13,7 @@
 int main(int argc, char *argv[]) {
   qInstallMessageHandler(swagLogMessageHandler);
   setpriority(PRIO_PROCESS, 0, -20);
+  util::set_core_affinity({6});
```

Comment:
this should go on the small cluster (0-3)

Reply:
done